### PR TITLE
feat: loader for past transactions, error alert for checkout

### PR DESCRIPTION
**Pull request by szenius**
This PR adds
* Spinner for checkout and no quota screens while past transactions are being fetched
* Error alert box for checkout screen when fetching past transactions fails

### CODE REVIEW

File: src/components/CustomerQuota/NoQuota/NoQuotaCard.tsx [hunk 1]
```diff
@@ -154,6 +154,12 @@ export const sortTransactions = (
     });
 };
 
+const getLatestTransactionTime = (quotaResponse: Quota) =>
```
Comment (szenius) [inline src/components/CustomerQuota/NoQuota/NoQuotaCard.tsx:157]: There is still a slight delay in loading latest transaction time in the title after making this change, so I am waiting for Pat and Eida to feedback before writing the unit test for this~

Comment (szenius) [reply src/components/CustomerQuota/NoQuota/NoQuotaCard.tsx]: added test le!!

**Review by sebastianquek — APPROVED** [review]
LGTM!! 👍

**Review by ilhaoni — APPROVED** [review]
LGTM! :D

**Pull request merged**